diff --git a/dist/index.d.mts b/dist/index.d.mts
index 4cfcb3a36457c79e914a6134ca1db00cc7a9d213..cadc7531b255f2a45cdf38ad7279c8d677b0243c 100644
--- a/dist/index.d.mts
+++ b/dist/index.d.mts
@@ -1,4 +1,4 @@
-import { State, AutocompletePrompt, Validate, DateFormat } from '@clack/core';
+import { State, AutocompletePrompt, Validate, DateFormat, CANCEL_SYMBOL } from '@clack/core';
 export { CANCEL_SYMBOL, ClackSettings, DateFormat, isCancel, settings, updateSettings } from '@clack/core';
 import { Readable, Writable } from 'node:stream';
 
@@ -103,7 +103,7 @@ interface SelectOptions<Value> extends CommonOptions {
      */
     showInstructions?: boolean;
 }
-declare const select: <Value>(opts: SelectOptions<Value>) => Promise<Value | symbol>;
+declare const select: <Value>(opts: SelectOptions<Value>) => Promise<Value | typeof CANCEL_SYMBOL>;
 
 /**
  * Options for the {@link autocomplete} prompt.
@@ -180,7 +180,7 @@ interface AutocompleteOptions<Value> extends AutocompleteSharedOptions<Value> {
  * });
  * ```
  */
-declare const autocomplete: <Value>(opts: AutocompleteOptions<Value>) => Promise<Value | symbol>;
+declare const autocomplete: <Value>(opts: AutocompleteOptions<Value>) => Promise<Value | typeof CANCEL_SYMBOL>;
 /**
  * Options for the {@link autocompleteMultiselect} prompt
  */
@@ -219,7 +219,7 @@ interface AutocompleteMultiSelectOptions<Value> extends AutocompleteSharedOption
  * });
  * ```
  */
-declare const autocompleteMultiselect: <Value>(opts: AutocompleteMultiSelectOptions<Value>) => Promise<Value[] | symbol>;
+declare const autocompleteMultiselect: <Value>(opts: AutocompleteMultiSelectOptions<Value>) => Promise<Value[] | typeof CANCEL_SYMBOL>;
 
 /**
  * Alignment for content or titles within the box.
@@ -332,7 +332,7 @@ interface ConfirmOptions extends CommonOptions {
  * });
  * ```
  */
-declare const confirm: (opts: ConfirmOptions) => Promise<boolean | symbol>;
+declare const confirm: (opts: ConfirmOptions) => Promise<boolean | typeof CANCEL_SYMBOL>;
 
 /**
  * Options for the {@link date} prompt.
@@ -396,7 +396,7 @@ interface DateOptions extends CommonOptions {
  * });
  * ```
  */
-declare const date: (opts: DateOptions) => Promise<Date | symbol>;
+declare const date: (opts: DateOptions) => Promise<Date | typeof CANCEL_SYMBOL>;
 
 type Prettify<T> = {
     [P in keyof T]: T[P];
@@ -527,7 +527,7 @@ interface GroupMultiSelectOptions<Value> extends CommonOptions {
  *
  * @param opts The options for the group multiselect prompt
  */
-declare const groupMultiselect: <Value>(opts: GroupMultiSelectOptions<Value>) => Promise<Value[] | symbol>;
+declare const groupMultiselect: <Value>(opts: GroupMultiSelectOptions<Value>) => Promise<Value[] | typeof CANCEL_SYMBOL>;
 
 /**
  * Options for the {@link limitOptions} function.
@@ -704,7 +704,7 @@ interface TextOptions extends CommonOptions {
  * });
  * ```
  */
-declare const text: (opts: TextOptions) => Promise<string | symbol>;
+declare const text: (opts: TextOptions) => Promise<string | typeof CANCEL_SYMBOL>;
 
 /**
  * Options for the {@link multiline} prompt
@@ -735,7 +735,7 @@ interface MultiLineOptions extends TextOptions {
  * });
  * ```
  */
-declare const multiline: (opts: MultiLineOptions) => Promise<string | symbol>;
+declare const multiline: (opts: MultiLineOptions) => Promise<string | typeof CANCEL_SYMBOL>;
 
 declare const MULTISELECT_INSTRUCTIONS: string[];
 interface MultiSelectOptions<Value> extends CommonOptions {
@@ -751,7 +751,7 @@ interface MultiSelectOptions<Value> extends CommonOptions {
      */
     showInstructions?: boolean;
 }
-declare const multiselect: <Value>(opts: MultiSelectOptions<Value>) => Promise<Value[] | symbol>;
+declare const multiselect: <Value>(opts: MultiSelectOptions<Value>) => Promise<Value[] | typeof CANCEL_SYMBOL>;
 
 type FormatFn = (line: string) => string;
 interface NoteOptions extends CommonOptions {
@@ -798,7 +798,7 @@ interface PasswordOptions extends CommonOptions {
  * });
  * ```
  */
-declare const password: (opts: PasswordOptions) => Promise<string | symbol>;
+declare const password: (opts: PasswordOptions) => Promise<string | typeof CANCEL_SYMBOL>;
 
 /**
  * Options for the {@link path} prompt.
@@ -848,7 +848,7 @@ interface PathOptions extends CommonOptions {
  * });
  * ```
  */
-declare const path: (opts: PathOptions) => Promise<string | symbol>;
+declare const path: (opts: PathOptions) => Promise<string | typeof CANCEL_SYMBOL>;
 
 interface SpinnerOptions extends CommonOptions {
     indicator?: 'dots' | 'timer';
@@ -886,7 +886,7 @@ interface SelectKeyOptions<Value extends string> extends CommonOptions {
     initialValue?: Value;
     caseSensitive?: boolean;
 }
-declare const selectKey: <Value extends string>(opts: SelectKeyOptions<Value>) => Promise<Value | symbol>;
+declare const selectKey: <Value extends string>(opts: SelectKeyOptions<Value>) => Promise<Value | typeof CANCEL_SYMBOL>;
 
 declare const stream: {
     message: (iterable: Iterable<string> | AsyncIterable<string>, { symbol }?: LogMessageOptions) => Promise<void>;
