diff --git a/dist/utilities/use-search-params/use-search-params.svelte.js b/dist/utilities/use-search-params/use-search-params.svelte.js
index a61b8d16429f7c1f04b8341f9553bdd47cb9f65b..0fd8f4017188313830c5c0cf1897078170a10fbb 100644
--- a/dist/utilities/use-search-params/use-search-params.svelte.js
+++ b/dist/utilities/use-search-params/use-search-params.svelte.js
@@ -1,7 +1,7 @@
 import { SvelteURLSearchParams } from "svelte/reactivity";
 import { dequal } from "dequal/lite";
 import * as lzString from "lz-string";
-import { browser, building } from "$app/environment";
+import { browser, building } from "$app/env";
 import { goto } from "$app/navigation";
 import { page } from "$app/state";
 import { IsMounted } from "../is-mounted/is-mounted.svelte.js";
